Gemfile

```diff
@@ -211,6 +211,7 @@ end
 
 group :test do
   # as test framework
+  gem 'sqlite3'
```

Comment:
Because of a strange issue i faced. When I tried to run 'bundle exec rake factory_girl:lint'  it shows me an error to include this gem. But when i tried "bundle exec rake factory_girl:lint RAILS_ENV='test'" there was no errors.
Though we check the ENV in factory_girl task but still i don't know why i got that error.